index.html

```diff
@@ -10183,6 +10183,7 @@ <h3>Value</h3>
 				<dd>A list of one or more tokens.</dd>
 			</dl>
 			<p>These are generic types for states and properties, but do not define specific representation. See <a href="#state_property_processing">State and Property Attribute Processing</a> for details on how these values are expressed and handled in host languages.</p>
+			<p>When the ARIA content attribute definition includes a table enumerating the attribute's allowed values, that attribute is an <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute">enumerated attribute</a>. When the values table notates one of the values as "(default)", that default value is the <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#missing-value-default">missing value default</a> and <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#invalid-value-default">invalid value default</a> for the attribute.</p>
```

Comment:
> When the ARIA content attribute definition includes a table enumerating the attribute's allowed values, that attribute is an enumerated attribute.

It'd be nice to give a couple of examples here, especially for enumerated "boolean" attributes, because imo this is  one of the confusing parts of ARIA spec, where an enumerated attribute has true/false values unlike HTML boolean attributes (compare aria-hidden and HTML:hidden). For example, something like this might make a good addition:
"For example, aria-readonly attribute is an enumerated attribute which can take true/false values or  aria-autocomplete which takes one of values from an enumerated list".

> When the values table notates one of the values as "(default)", that default value is the missing value default and invalid value default for the attribute.

I might be wrong but "missing value default" and "invalid value default" are mutually exclusive, i.e. default value can be either "missing value default" or "invalid value default", if so then "and" should be replaced on "or". Also it'd be good to give a couple of examples of attributes having default value for each case.

Reply:
Missing value default and invalid value default are not mutually exclusive; the same value can be both.

+1 to examples.

Reply:
@asurkov wrote:

> It'd be nice to give a couple of examples here, especially for enumerated "boolean" attributes 

Agreed. I'll add some.